app/services/search_service.rb

```diff
@@ -15,10 +15,11 @@ def filter_on_search_string(base_query, params)
 
       params[:search_string].split.reduce(base_query) do |built_query, token|
         word = token.strip
```

Comment:
The other thing you did on the other PR was to swap `'` for `''`. I think we could stand to do that here, otherwise it breaks if you search for "O'Reilly"

Reply:
Oh yes so I did